### ISSUE TITLE
1.10.2 update will not start requires 1.11

### ISSUE THREAD

**Issue by jprocket45** [Chisel-Team/Chisel#371]
The new update threw curse was submitted for 1.10.2 it won't start up due to it says it requires 1.11 or 1.12??? 

Ver 0.0.9.9

![image](https://cloud.githubusercontent.com/assets/11837257/23510974/9fd3f47c-ff29-11e6-8533-27fbd5dd3830.png)

**Comment by Winddbourne** [comment]
I'm having this issue too.  I checked the file and it is Chisel - Mc1.11.2-0.0.9.9 if I'm understanding this is supposed to be a 1.11 update which curse is somehow assuming also supports 1.10.2 and giving us mod authors an update arrow for on our custom pack work page. 

You might want to check to see if it is an ALSO not a rather . . . because 1.11.2  may not have gotten their update today. lol. In the meantime I've rolled back the file to the last 1.10.2 version and I'm making a mental note not to hit the update arrow next time I'm upgrading my files to the latest version.
